modules/serverless-rn-1-20-0.adoc

```diff
@@ -0,0 +1,41 @@
+[id="serverless-rn-1-20-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.20.0
+
+[id="new-features-1-20-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.26.
+* {ServerlessProductName} now uses Knative Eventing 0.26.
+* {ServerlessProductName} now uses Kourier 0.26.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.26.
+* {ServerlessProductName} now uses Knative Kafka 0.26.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+// Is `kn func` using version later than 0.19?
+
+* The Kafka broker is now available as a Technology Preview.
+
+[id="fixed-issues-1-20-0_{context}"]
+== Fixed issues
+
+[id="known-issues-1-20-0_{context}"]
+== Known issues
+
+* {ServerlessProductName} deploys kservices with the default address that uses HTTPS. When using the in-cluster sender job, the sender does not have the cluster Certificate Authority (CA) configured.
++
+Consequently, the delivery fails, unless cluster uses globally accepted Certificates.
++
+For example, this delivery works:
++
+[source,terminal]
+----
+$ kn event send --to-url https://sockeye-default.openshift.org
```

Comment:
Maybe we should use standard `example.org` sub-domain.
```suggestion
$ kn event send --to-url https://sockeye-default.openshift.example.org
```

Reply:
It would be nice if this used the same URL used in older docs, e.g. https://deploy-preview-40821--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-applications.html#kn-service-describe_serverless-applications

`http://hello-default.apps.ocp.example.com`
`http://event-delivery-default.example.com`
something like this